File: BinaryTree/Leetcode102.java
public class Leetcode102 {
    public List<List<Integer>> levelOrder(TreeNode root) {
        List<List<Integer>> res = new ArrayList<>();
        if(root == null)return res;
        Queue<TreeNode> q = new LinkedList<>();
        q.offer(root);
        while(!q.isEmpty()){
            int size = q.size();
            List<Integer> level = new ArrayList<>();
            for(int i = 0;i < size;i++){
                TreeNode n = q.poll();
                if(n.left != null)q.offer(n.left);
                if(n.right != null)q.offer(n.right);
                level.add(n.val);
            }
            res.add(new ArrayList<>(level));
        }
        return res;
}

File: BinarySearchTree/Python/Leetcode102.py
from queue import Queue
class Leetcode102:
    def levelOrder(self, root: TreeNode) -> List[List[int]]:
        res = []
        if root is None:
            return res
        q = Queue()
        q.put(root)
        while not q.empty():
            levelNode = []
            for i in range(q.qsize()):
                n = q.get()
                if n.left is not None: q.put(n.left)
                if n.right is not None: q.put(n.right)
                levelNode.append(n.val)
            res.append(levelNode)
        return res
